src/main/scala/com/typesafe/sbt/packager/rpm/RpmMetadata.scala

```diff
@@ -267,17 +264,15 @@ case class RpmSpec(
     sb append "\n\n"
 
     // write build as moving everything into RPM directory.
-    sb append installSection(tmpRoot)
+    sb append installSection(tmpRoot, buildSymlinkScript(meta.name, installDir, symlinks))
     // TODO - Allow symlinks
 
     // write scriptlets
     sb append scriptlets.pretransContent()
     sb append scriptlets.preContent()
-    sb append scriptlets.postContent(buildSymlinkScript(meta.name, installDir, symlinks))
```

Comment:
Actually yes. If I understood the rpm docs correctly, this should be handled by the `%install` section.